packages/expo-image/src/Image.tsx

```diff
@@ -199,13 +199,6 @@ export class Image extends React.PureComponent<ImageProps> {
       loggedDefaultSourceDeprecationWarning = true;
     }
 
-    if (restProps.children && !loggedRenderingChildrenWarning) {
-      console.warn(
-        'The <Image> component does not support children. If you want to render content on top of the image, consider using the <ImageBackground> component or absolute positioning.'
-      );
-      loggedRenderingChildrenWarning = true;
-    }
```

Comment:
I think we don't want to remove that warning – it's good to let the user know if he is doing something wrong

Reply:
When building Expo-Image, Omitting the children will cause a build failure. This initial logger was setup due to a discord discussion we had when it wasn't clear you could not wrap children in an image component. By type hinting this error should be apparent. I could rebuild throwing a @ts-ignore to build this. What do you think?

Reply:
In development Metro bundler will not complain if you have issues in types, it strips out all the types. Your code editor will hint you about the mismatch in types, but I think this is not enough. Especially when your app is big, you don't immediately see all such issues when you upgrade the package until the component is used.